packages/picasso.js/src/core/chart-components/ref-line/index.js

```diff
@@ -14,6 +14,10 @@ import refLineComponent from './refline';
  *      label: {
  *        text: 'value label'
  *      }
+ *      slope: {
```

Comment:
What would happend if put this under the `x` line instead?

Reply:
slope is only created for y - axis now